blocks/models/huggingface/base.py

```diff
@@ -89,28 +104,28 @@ def load(self) -> None:
             LoadOrigin.pretrained: self.config.pretrained_model,
         }
 
-        load_order = [
-            (
-                self.config.preferred_load_origin,
-                paths[self.config.preferred_load_origin],
-            )
-        ] + [
-            (key, path)
-            for key, path in paths.items()
-            if path is not paths[self.config.preferred_load_origin]
-        ]
+        if (
+            hasattr(self.config, "preferred_load_origin")
+            and self.config.preferred_load_origin is not None
+        ):
+            load_order = [
+                (
+                    self.config.preferred_load_origin,
+                    paths[self.config.preferred_load_origin],
+                )
+            ]
+        else:
+            load_order = [(origin, path) for origin, path in paths.items()]
```

Comment:
isn't this equivalent to load_order = paths.items()?

Reply:
Yes 🤦

Reply:
Fixed.